common/djangoapps/student/urls.py

```diff
@@ -38,27 +47,18 @@
         url(r'^auto_auth$', 'auto_auth'),
     )
 
+# add all student.views url patterns
 urlpatterns = patterns(*urlpatterns)
 
-# password reset mixes student.views with django views
-urlpatterns += (
-    url(r'^password_reset/$', 'student.views.password_reset', name='password_reset'),
 
-    # Obsolete Django views for password resets
+# password reset django views (see above for password reset student.views)
+urlpatterns += patterns(
+    'django.contrib.auth.views',
 
     # TODO: Replace with Mako-ized views
-
-    url(r'^password_change/$', 'django.contrib.auth.views.password_change', name='password_change'),
```

Comment:
Those may have been from the old login-and-registration flow: @rlucioni, I summon you!